src/main/php/com/amazon/aws/credentials/Provider.class.php

```diff
@@ -1,8 +1,10 @@
 <?php namespace com\amazon\aws\credentials;
 
-interface Provider {
+abstract class Provider {
```

Comment:
This is a BC break and Rezepte s major version release plus cascading dependency updates. I really wish we had interface default methods...